metadata_tools/common.py

```
@@ -0,0 +1,224 @@
+SYSTEM_NULL = "NONE"
+
+#=========================================================================================
+def init_logger(dir, type):
+    """Initialize logger.
+    Args:
+        dir (FCPath): Directory to log.
+        type (str): Type of log to create.
+    Returns:
+        None
+    """
+    name = '%s_%s-log.txt' % (dir.name, type)
```

Ruff (0.14.10)

Use format specifiers instead of percent format

Replace with format specifiers

(UP031)